pkgs/applications/emulators/86box/default.nix

```diff
@@ -0,0 +1,74 @@
+{ stdenv, lib, fetchFromGitHub, cmake, pkg-config, makeWrapper, freetype, SDL2
+, glib, pcre2, openal, rtmidi, fluidsynth, jack2, alsa-lib, qt5, libvncserver
+, discord-gamesdk, libpcap
+
+, enableDynarec ? with stdenv; isx86_32 || isx86_64 || isAarch64 || isAarch32
+, enableNewDynarec ? with stdenv; enableDynarec && (isAarch64 || isAarch32)
```

Comment:
```suggestion
, enableNewDynarec ? enableDynarec && stdenv.isAarch
```

Reply:
Neither `stdenv.isAarch` or `stdenv.isx86` seem to exist; apparently, those attrs are only available on `stdenv.hostPlatform`, so I moved to using `hostPlatform` instead.

Reply:
Oh, yes, good catch but I think you got my general direction.